business_objects/upload_task.py

```diff
@@ -1,5 +1,5 @@
 import datetime
-from typing import Optional, Dict, Any
+from typing import ByteString, Optional, Dict, Any
```

Comment:
is ByteString a typing import?

Reply:
yes, I searched for Byte initially but seems like ByteString is the valid option python has

Reply:
makes sense, however, it's not used here

Reply:
Will remove it

Reply:
Added it as type for key